src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html

```diff
@@ -14,10 +14,9 @@
        href="{{ docsUrl }}"
        target="_blank"
        i18n>documentation</a>
-    <a ngbDropdownItem
-       href="/docs"
-       target="_blank"
-       i18n>API</a>
+    <button ngbDropdownItem
+            (click)="navigate()"
```

Comment:
I refactored this recently..I think we don't need button here. You can take a look [here](https://github.com/ceph/ceph/pull/41430/files)

Reply:
As discussed with you, current approach is correct. Thanks